ivadomed/transforms.py

```diff
@@ -335,7 +338,7 @@ def __call__(self, sample, metadata):
         # Get params
         is_2d = sample.shape[-1] == 1
         th, tw, td = self.size
-        fh, fw, fd, h, w, d = metadata['crop_params']
+        fh, fw, fd, h, w, d = metadata['crop_params'][self.__class__.__name__]
```

Comment:
Exactly :), currently for 2D images with object detection, we need to apply a CenterCrop after the BoundingBoxCrop to ensure all batches have the same size (since we don't have patch-wise training in 2D). So I added this to accommodate this need.